WordPressAuthenticator/NUX/NUXLinkMailViewController.swift

```diff
@@ -75,14 +75,8 @@ class NUXLinkMailViewController: LoginViewController {
 
         usePasswordButton?.isHidden = emailMagicLinkSource == .signup
 
-        label?.text = {
-            switch emailMagicLinkSource {
-            case .login:
-                return NSLocalizedString("Your magic link is on its way! Check your email on this device, and tap the link in the email you receive from WordPress.com", comment: "Instructional text on how to open the email containing a magic link.")
-            case .signup:
-                return NSLocalizedString("We sent you a magic signup link! Check your email on this device, and tap the link in the email to finish signing up.", comment: "Instructional text on how to open the email containing a magic link.")
-            }
-        }()
+        label?.text = NSLocalizedString("Check your email on this device, and tap the link in the email you receive from WordPress.com.\n\nNot seeing the email? Check your Spam or Junk Mail folder.", comment: "Instructional text on how to open the email containing a magic link.")
```

Comment:
There is a spelling error on the design. It should say `received` instead of `receive`. Would you mind updating it here?

Reply:
Hrm. It makes sense to me either way; I guess it depends on how you read it. 🤷‍♂ But sure, I can change it.